lib/token_authenticate_me/concerns/controllers/password_resetable.rb

```diff
@@ -16,7 +16,7 @@ module PasswordResetable
 
           # Send reset token to user with e-mail address
           def create
-            @user = User.find_by_email(params[:email])
+            @user = User.find_by_email(email)
```

Comment:
We should convert this to `find_by(find_by_hash)` that way it is even more flexible.